stack.yaml

```diff
@@ -1,3 +1,3 @@
-resolver: lts-17.1
+resolver: https://raw.githubusercontent.com/freckle/stackage-snapshots/main/lts/18/10.yaml
```

Comment:
I think we decided not to do this. Our custom snapshots are for our applications. Libraries, in the spirit of easing outside collaboration as much as possible, shouldn't couple so tightly to our app ecosystem through this.

It also sets up this weird circular thing where this library is both in, and depends on, our snapshot.